pages/one-page-reduced-address.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Footer from '../components/footer.jsx';
+import Header from '../components/header.jsx';
+import SmallPrint from '../components/small-print.jsx';
+import SingleForm from '../components/single-form.jsx';
+
+var OnePage = React.createClass({
+  mixins: [require('react-intl').IntlMixin],
+  render: function() {
+    var className = "row new-flow-test";
+    if (this.props.test) {
+      className += " " + this.props.test;
+    }
+    return (
+      <div className={className}>
+        <Header locale={this.props.locales[0]} alt={this.getIntlMessage('donate_to_mozilla')}></Header>
+        <SingleForm
+          billingAddress={false}
```

Comment:
We have another patch for this, sorry for the lack of communication.

However, for next time, there is still some query params we would need to remove and ensure the old page passed in true.

You can see the landed patch here: https://github.com/mozilla/donate.mozilla.org/commit/3ca57ba5a1f2d580ca66eb8058cf231cc8f94cfc

Again, sorry about that.